portfolio/src/main/webapp/frontPage.html

```diff
@@ -41,20 +41,23 @@ <h1>My Gallery</h1>
       <div id="greeting-container"></div>
       <br>
       <br>
-      <h1>Comment</h1>
-      <br>
-      <p>Number of comments displayed: </p>
-      <input type="number" id="numberOfComments" value="0" min="0" onchange="loadComments()">
-      <br>
-      <p>Click on comment to delete it</p>
-      <br>
-      <form action='/comments' method="POST">
-        <textarea name="comment"></textarea>
-	<br>
-	<input type="submit"/>
-      </form>
-      <br>
-      <ul id="comment-section"></ul>
+      <div id="display-comments">
+	<h1>Comments</h1>
+        <br>
+        <p>Number of comments displayed: </p>
+        <input type="number" id="numberOfComments" value="0" min="0" onchange="loadComments()">
+        <br>
+        <p>Click on comment to delete it</p>
+        <br>
+        <form action='/comments' method="POST">
+          <textarea name="comment"></textarea>
+	  <br>
+	  <input type="submit"/>
```

Comment:
Please indent everything within the form.

Reply:
Ok